subsystem-test/framework/src/main/java/org/jboss/as/subsystem/test/AbstractSubsystemBaseTest.java

```diff
@@ -163,6 +164,7 @@ public void testSchema() throws Exception {
      * To enable a test for validation of the subsystem templates, override both {@link #getSubsystemXsdPath()} and {@link #getSubsystemTemplatePaths()}
      * then add a new test just calling this method.
      */
+    @Deprecated
     protected void testSchemaOfSubsystemTemplates() throws Exception {
```

Comment:
@lvydra refers to the Jira description; testSchemaOfSubsystemTemplates turned into something that just throws AssumptionViolationException.